src/Relation/Binary/Core.agda

```diff
@@ -52,6 +52,13 @@ infixr 4 _⇒_ _=[_]⇒_
 _⇒_ : REL A B ℓ₁ → REL A B ℓ₂ → Set _
 P ⇒ Q = ∀ {i j} → P i j → Q i j
 
+
+infix 3 _←→_
+
+_←→_ :  Set a → Set b → Set _   -- "if and only if"
```

Comment:
This new type isn't actually needed in this PR so please can you remove it for the moment. Anyway this is pretty close to `_⇔_` in `Function.Equivalence` so it might be able to be replaced later.

Reply:
Removed.